app/models/report.rb

```diff
@@ -49,7 +49,7 @@ class Report < ActiveRecord::Base
   scope :completed, -> { where(status: %w(completed archived)) }
 
   scope :pending, lambda {
-    joins(:dispatches).where(dispatches: {status: 'pending'})
+    joins(:dispatches).where(dispatches: {status: 'pending'}).distinct
```

Comment:
The way we track the status of multiple responders for a single report is by a record in the dispatches table for the status of each responders for a given report matched the report as a foreign key. We want to return a single copy of each report, with the dispatches records joined to it. We think distinct provides the correct behavior.